src/components/pages/proof-of-reserves.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState, useEffect } from "react"
+import axios from "axios"
+import ValidateProof from "components/proof-of-reserves/proof"
+import Header from "components/header"
+import { NoPropsFCT, useAuthContext } from "store"
+import useMainQuery from "hooks/use-main-query"
+
+const ListInputField: NoPropsFCT = () => {
+  const [rootHash, setRootHash] = useState("")
+  const [data, setData] = useState(null)
+  const [error, setError] = useState(null)
+  const { btcWalletId } = useMainQuery()
+
+  const fetchTreeMetadata = async () => {
+    const query = `query TreeMetadata{
+      treeMetadata{
+        roothash
+      }
+    }`
+    const response = await axios({
+      url: "http://localhost:4004/",
+      method: "post",
+      data: {
+        query,
+      },
+    })
```

Comment:
URL needs to be in an env var, and the request should use a useQuery hook (from galoy-client)